### PR TITLE
enhance: add restful api to trigger component stop

### PR DESCRIPTION
**Pull request by weiliu1031**
issue: #32698
This PR add two rest api for component stop and status check:
1. `/management/stop?role=querynode` can stop the specified component
2. `/management/check/ready?role=rootcoord` can check whether the target component is serviceable

### CODE REVIEW

File: internal/http/router.go [hunk 1]
```diff
@@ -27,3 +27,9 @@ const EventLogRouterPath = "/eventlog"
 
 // ExprPath is path for expression.
 const ExprPath = "/expr"
+
+// TriggerStopPath is path for trigger component stop
+const TriggerStopPath = "/stop"
```
Comment (jaime0815) [inline internal/http/router.go:32]: suggest moving all operations of cluster management to /management root path,  we should be cautious while add the kind of path under root path

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**